src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -457,6 +458,13 @@ record = _serverStore.Cluster.ReadDatabase(context, Name, out index);
                         _logger);
                     try
                     {
+                        using (DocumentsStorage.ContextPool.AllocateOperationContext(out DocumentsOperationContext ctx))
+                        using (ctx.OpenReadTransaction())
+                        {
+                            var lastCompletedClusterTransactionIndex = DocumentsStorage.ReadLastCompletedClusterTransactionIndex(ctx.Transaction.InnerTransaction);
+                            Interlocked.Exchange(ref LastCompletedClusterTransactionIndex, lastCompletedClusterTransactionIndex);
+                        }
```

Comment:
Why are you doing that in the background thread?
This is a one time operation that should go in the database init. 
Then you don't need to worry about concurrency

Reply:
Done